movement/do-movement.scm

```diff
@@ -0,0 +1,205 @@
+;
+; do-movement.scm
+;
+; Implement the movement API for ROS/blender animations.
+;
+
+; Delete the current definition, if any.
+(define (delete-definition STR)
```

Comment:
DefineLink definitions cannot ordinarily be re-defined.  However, here, we want to have two different definitions of these DefinedPredicateNodes: one version which talks to ROS, and one version which does not.  If the non-ROS version is loaded first, and then the ROS version is loaded later, the first set of DefineLinks need to be deleted.

There are other ways of solving this problem, too. This seemed like ... one of the better ones.

Another solution would be to provide two python versions: one with and one without ROS (i.e. by duck-typing the python code, instead of duck-typing the DefinedPredicateLinks)  This had the disadvantage of forcing the use of python; instead, it seemed like the duck-typed DefinedPredicateLinks would be better, as maybe some future version could use ... c++, haskell, or atomese, or whatever, to define these.